MapleServer2/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "MapleServer2": {
+      "commandName": "Project",
+      "workingDirectory": "E:\\MS2 Pserver\\MapleStory 2 Server Emulator"
```

Comment:
Can you change the working directory to `$(SolutionDir)`
It should fix the path problem. Or add this file to gitignore in the future :D

Reply:
This file isn't included in the PR where did this come from?

Reply:
This file is on my end. thats why its deleted off this PR